src/app/components/list/view-switcher/index.html

```diff
@@ -21,44 +21,44 @@
       <h3 class="sky-text-success">Use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Use the list view switcher when the user needs to switch between different views of the same data.</p>
+              <p>Use list view switchers when users need to switch between different views of the same data.</p>
```

Comment:
@Blackbaud-ToddRoberts, should we provide an example of when/why multiple views are necessary to provide some context for the use cases of this component?

Reply:
added

Reply:
LGTM. Thanks!